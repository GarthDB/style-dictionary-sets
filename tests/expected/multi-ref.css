:root {
  --component-background-color: var(--color-gray-100);
  --static-colors-light-gray: #EEEEEE;
  --static-colors-dark-gray: #111111;
}
