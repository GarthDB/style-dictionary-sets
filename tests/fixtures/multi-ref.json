{
  "component": {
    "background-color": {
      "value": "{color.gray.100}"
    }
  },
  "color": {
    "gray": {
      "100": {
        "sets": {
          "light": {
            "value": "{static-colors.light-gray}"
          },
          "dark": {
            "value": "{static-colors.dark-gray}"
          }
        }
      }
    }
  },
  "static-colors": {
    "light-gray": { "value": "#EEEEEE" },
    "dark-gray": { "value": "#111111" }
  }
}
