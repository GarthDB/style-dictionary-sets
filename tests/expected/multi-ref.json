{
  "component": {
    "background-color": {
      "sets": {
        "ref": "{color.gray.100}",
        "sets": {
          "light": {
            "value": "#EEEEEE"
          },
          "dark": {
            "value": "#111111"
          }
        }
      }
    }
  },
  "color": {
    "gray": {
      "100": {
        "sets": {
          "light": {
            "value": "#EEEEEE"
          },
          "dark": {
            "value": "#111111"
          }
        }
      }
    }
  },
  "static-colors": {
    "light-gray": {
      "value": "#EEEEEE"
    },
    "dark-gray": {
      "value": "#111111"
    }
  }
}
